schemas/IDLs/org.gel.models.report.avro/6.0.0/ClinicalReportRD.avdl

```diff
@@ -0,0 +1,84 @@
+@namespace("org.gel.models.report.avro")
```

Comment:
```
$ diff 5.0.0/ClinicalReportRD.avdl 6.0.0/ClinicalReportRD.avdl 
55c55
<         union {null, array<ReportedVariant>} variants;
---
>         union {null, array<SmallVariant>} variants;
```

Reply:
could you ellaborate @greglever ? words are free!

Reply:
@priesgo It's just to show that the only difference between this model in 5 and 6 is that `variants` is now an array of `SmallVariant` objects instead of `ReportedVariant` objects

Reply:
I see, I thought you were requesting a change there.